Latest python37Packages.altair fails on build due to urllib test failure

**Describe the bug**

On the latest nixpkgs revision 0d97ef510bdc9d66f1023f970be58fdab2eb87fa, building altair fails:

```
nix-repl> :b pkgs.python37Packages.altair
builder for '/nix/store/h5s87k67szlaraffzqlpj1nvrw7drk1a-python3.7-altair-4.0.0.drv' failed with exit code 2; last 10 log lines:
      raise URLError(err)
  E   urllib.error.URLError: <urlopen error [Errno -2] Name or service not known>
  =============================== warnings summary ===============================
  /nix/store/k7iah7fy3hkcavczk413x6c9wiqkqli8-python3.7-ipython-7.10.2/lib/python3.7/site-packages/IPython/paths.py:68
    /nix/store/k7iah7fy3hkcavczk413x6c9wiqkqli8-python3.7-ipython-7.10.2/lib/python3.7/site-packages/IPython/paths.py:68: UserWarning: IPython parent '/homeless-shelter' is not a writable location, using a temp directory.
      " using a temp directory.".format(parent))
  
  -- Docs: https://docs.pytest.org/en/latest/warnings.html
  !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
  ========================= 1 warning, 1 error in 7.84s ==========================
[0 built (1 failed), 122 copied (688.7 MiB), 114.1 MiB DL]
error: build of '/nix/store/h5s87k67szlaraffzqlpj1nvrw7drk1a-python3.7-altair-4.0.0.drv' failed
```

--- a/pkgs/development/python-modules/altair/default.nix
+++ b/pkgs/development/python-modules/altair/default.nix
@@ -47,6 +47,8 @@ buildPythonPackage rec {
 
   checkPhase = ''
     export LANG=en_US.UTF-8
+    # histogram_responsive.py attempt network access, and cannot be disabled through pytest flags
+    rm altair/examples/histogram_responsive.py
     pytest --doctest-modules altair
   '';
 

